workflows/dragen-somatic-with-germline-pipeline/4.2.4/dragen-somatic-with-germline-pipeline__4.2.4.cwl

```diff
@@ -923,6 +923,23 @@ inputs:
       - pattern: ".tbi"
         required: true
 
+  # Cross sample contamination
+  qc_cross_cont_vcf:
+    label: qc cross cont vcf
+    doc: |
+      The cross-contamination metric is enabled by including one of the following flags along with a compatible VCF.
+      Pre-built contamination VCF files for different human references can be found at /opt/edico/config. 
+      DRAGEN supports separate modes for germline and somatic samples. 
+    type: File?
```

Comment:
How do I specify a string input if the only type option provided at the workflow level is a file type?

Reply:
Enum option still needs to be provided here